src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -146,9 +146,12 @@ export class CalciteCombobox {
       case "Tab":
         this.activeChipIndex = -1;
         this.activeItemIndex = -1;
-        this.active = false;
         if (this.allowCustomValues && this.text) {
           this.addCustomChip(this.text, true);
+          event.preventDefault();
+          event.stopPropagation();
```

Comment:
Can this be done w/o stopping propagation? For example, a dev may be listening for all global key presses and this would break that logic.

Reply:
mm, yes I think I only really need prevent default here, will update